const v2 = new AggregateError(3);
function f9(a10, a11) {
    const o21 = {
        set f(a13) {
        },
        [1073741823](a18, a19) {
            AggregateError.prototype **= -9007199254740992;
            try {
                this.f = a19;
            } catch(e20) {
            }
            return a11;
        },
    };
    return o21;
}
f9(3, v2);
f9(4294967296, -1268);
f9(-1268, 536870887);
function f27(a28, a29, a30, a31) {
    a28.prototype = a28;
    try { a28(a31); } catch (e) {}
    a29 & Uint16Array;
    const o37 = {
        n(a35, a36) {
            return a31;
        },
    };
    return Uint8Array;
}
for (let v38 = 0; v38 < 500; v38++) {
    f27(Uint8Array);
}
